docs/src/pages/components/data-grid/editing/editing.md

```diff
@@ -7,6 +7,33 @@ components: DataGrid, XGrid
 
 <p class="description">Data Grid will provide full support to create, read, update, and delete operations (CRUD).</p>
 
+## Cell editing
+
+Cell editing allows to edit the value of one cell at the time.
+To activate editing on cells, set the 'editable' property of the column definition `GridColDef` object to true.
+
+```tsx
+<DataGrid columns={[{ field: 'name', editable: true }]} />
+```
+
+By default, a cell can turn into edit mode by double clicking on it, or using <kbd>ENTER</kbd> key, if the cell has focus.
+
+{{"demo": "pages/components/data-grid/editing/BasicEditingGrid.js", "bg": "inline"}}
+
+### Control cell editability with isCellEditable
```

Comment:
The title is too long (I believe), see rule 3. of https://www.oreilly.com/content/the-eight-rules-of-good-documentation/

```suggestion
### Control cell editability
```